portfolio/src/main/webapp/script.js

```diff
@@ -77,7 +77,7 @@ async function fetchFromData() {
 /**
  * Creates a comment element with the content, name of commenter, and date
```

Comment:
Seems like this comment should be updated

Reply:
good catch! thank yoU!